### PR TITLE
chore(k8s): Extract common definitions

### PR DESCRIPTION
**Pull request by candiduslynx**

### CODE REVIEW

**Comment by candiduslynx** [comment]
@disq I've just updated the code to use explicit column definition in 1 table to unblock from waiting on https://github.com/cloudquery/plugin-sdk/pull/739.
I'll update the code in a separate PR once the plugin sdk PR is merged & released.

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**
